cmd/snappy/cmd_update.go

```diff
@@ -53,7 +53,7 @@ const (
 	shutdownTimeout = "+10"
 )
 
-var shutdownMsg = i18n.G("snappy autopilot triggered a reboot to boot into an up to date system -- temprorarily disable the reboot by running 'sudo shutdown -c'")
```

Comment:
Are you going to update the branch to bring this in, John, or is your intention to do that later?